src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppBase.cs

```diff
@@ -267,7 +285,17 @@ private async Task GenerateAndStoreReceiptPDF(Instance instance, DataElement dat
             UserContext userContext = await _userHelper.GetUserContext(_httpContextAccessor.HttpContext);
             UserProfile userProfile = await _profileService.GetUserProfile(userContext.UserId);
 
-            string formLayoutsString = _resourceService.GetLayouts();
+            string formLayoutsString = null;
+
+            if (layoutSet == null)
```

Comment:
Why not use ternary operator?

Reply:
Personal i belive i less readable, but I have updated it